src/pages/Firstbook.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable arrow-body-style */
+
```

Comment:
- [ ] `required`
- [ ] Customizing the Linters by `Disabling the eslint` defeats the purpose of using it please enable it.

Reply:
@mwanawabangona here the same